Mathlib/Analysis/Normed/Group/AddTorsor.lean

```diff
@@ -210,9 +212,12 @@ theorem LipschitzWith.vadd [PseudoEMetricSpace α] {f : α → V} {g : α → P}
       edist_vadd_vadd_le _ _ _ _
     _ ≤ Kf * edist x y + Kg * edist x y := (add_le_add (hf x y) (hg x y))
     _ = (Kf + Kg) * edist x y := (add_mul _ _ _).symm
-    
+
 #align lipschitz_with.vadd LipschitzWith.vadd
 
+-- Porting note: lean fails to find this instance
+instance Function.instAddTorsor : AddTorsor (α → V) (α → P) := Pi.instAddTorsorForAllForAllAddGroup
```

Comment:
If I remove this everything keeps working.

Reply:
In fact, `instance Function.instAddTorsor : AddTorsor (α → V) (α → P) := by infer_instance` now succeeds; so I think this line can be removed?

Reply:
Indeed the extra instance can be entirely removed.

There are still two `simp` replacements without corresponding porting notes.

Reply:
I would prefer to wait for @mcdoll to confirm whether there was a other reason for this instance

Reply:
> There are still two simp replacements without corresponding porting notes.

can you add PR comments on these to draw attention to them?

Reply:
:arrow_down:

Reply:
Sorry, I missed your ping Eric. I did not add any porting notes, since I don't think these proofs are necessarily worse than the original ones, but if you want to have them, that's fine with me too.